xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -6910,22 +6910,22 @@ init(xlator_t *this_xl)
     /* What you need to find with we do 'df -h' */
     cmd_args = &this_xl->ctx->cmd_args;
     if (cmd_args->fs_display_name) {
-        strcpy(fsname, cmd_args->fs_display_name);
+        strncpy(fsname, cmd_args->fs_display_name, PATH_MAX);
```

Comment:
`strncpy()` doesn't ensure null termination if source string is longer than `PATH_MAX`. It also fills the remaining space in the array with 0's if the source string is shorter, which is useless.